src/tribol/geom/ContactPlane.hpp

```diff
@@ -30,15 +30,17 @@ class ContactPlaneManager;
  * \param [in] pair interface pair containing pair related indices
  * \param [in] cMethod the Tribol contact method
  * \param [in] cCase the Tribol contact Case
+ * \param [in] inContact true if pair are in contact per CG routines
  *
- * \return true if face-pair are interacting
+ * \return 0 if no error, 1 if error
```

Comment:
Note: This is a breaking change that should probably be mentioned in the `RELEASE-NOTES`. I think it's mostly an internal function call, so it probably won't impact that much.

Reply:
Updated.